pkg/render/node_test.go

```diff
@@ -199,6 +199,8 @@ var _ = Describe("Node rendering tests", func() {
   "plugins": [
     {
       "type": "calico",
+	  "policy_setup_timeout_seconds": 0,
```

Comment:
Indentation is wrong here and doesn't match the other fields

Reply:
VSCode acting up, thanks

Reply:
Found a few other occurrences of tabs instead of spaces in this file's raw json strings, so fixed those up too.